app.yaml

```diff
@@ -62,6 +62,10 @@ handlers:
   # TODO(Sean Lip): Add cache when system to break cache during
   # new release is figured out.
     Cache-Control: 'no-cache'
+- url: /extensions
+  static_dir: extensions
+  application_readable: true
```

Comment:
OK -- but then, I think, per what Ben, you and I discussed in our last meeting, we cannot merge this until that final state of affairs is completed.